user_manual/android_app.rst

```diff
@@ -2,202 +2,305 @@
+connect via ``https``. 
+The ownCloud app will test your connection as soon as you provide it and tell
+you if you entered it correctly. 
+
+If your server has a self-signed SSL certificate, you'll get a warning that it
+is not to be trusted. 
+If this happens, click the OK button to accept the certificate and complete
+your account setup.
 
 .. figure:: images/android-3.png 
    :alt: SSL certificate warning.
 
-Managing Files
---------------
+With that completed, you're now ready to use the Android application. 
+At this point, you'll be on the All files screen, which you see below.
 
-Now you should see the Files page of your ownCloud account. Note some 
-differences from the previous release: click the blue button at the bottom 
-right to expose the **Upload**, **Content from other apps**, and **New folder** 
-buttons.
+.. figure:: images/android-all-files-overview.png 
+   :alt: All files screen.
 
-Use the **Upload** button to add files to your ownCloud account from your 
-Android filesystem. Use **Content from other apps** to upload files from 
-Android apps, such as the Gallery app.
+By clicking the main menu at the top left, you will be able to manage the core functionality of the app. The options are:
 
-.. figure:: images/android-4.png 
-   :alt: Your ownCloud Files page.
-   
-Click the overflow button at the top right (that's the one with three vertical 
-dots) to open a user menu. **Grid view** toggles between grid and list 
-view. **Refresh account** syncs with the server, and **Sort** 
-gives you the option to sort your files by date, or alphabetically.
+- Manage Users Accounts
+- Current **Uploads** 
+- All Files
+- Application **Settings**
+
+Manage Users Accounts
+~~~~~~~~~~~~~~~~~~~~~
+
+Initially the path to this section isn't visible. 
+To get to it, first click the down arrow, in the user details section, which
+will replace the "All files" and "Uploads" buttons with "Add account" and
+"Manage accounts". 
+Then, click "Manage accounts". 
+From there, you can see all of the currently active user accounts, along with a button to add a new account.
+
+User Accounts
+^^^^^^^^^^^^^
+
+After clicking "Manage Accounts", you will see a list of the currently active
+accounts in the application, as in the screenshot below. 
+Each entry in the list has shortcuts to:
+
+- Viewing the user's files
+- Changing the user's password
+- Removing the account
 
 .. figure:: images/android-6.png
    :alt: Top-right menu.   
 
-The ownCloud menu at the top left shows the name of the logged-in user, has a 
-shortcut to your files, the **Uploads** button, and the **Settings** button.
+Change User Password
+^^^^^^^^^^^^^^^^^^^^
+
+To change a user's password, click the key icon, next to the user's details. 
+This will display the user details page, with the ownCloud server URI and user account pre-filled.
+Enter a new password, and click "Connect", and the password will be updated.
+
+.. NOTE:: 
```

Comment:
How about moving the documentation there, but still referencing it here as well?